web_export_view/view/web_export_view.xml

```diff
@@ -2,7 +2,7 @@
 
 <openerp>
     <data>
-        <template id="assets_backend" name="web_export_view assets" inherit_id="web.assets_backend">
+        <template id="assets_backend" name="web_export_view_assets" inherit_id="web.assets_backend">
```

Comment:
Well, it is an arbitrary change but it is non blocking for me. It does not impede inheritance because inheritance should refer to the template id, not the name.